000-debug/logger.php

```diff
@@ -205,7 +208,7 @@ function vip_timer( $name = '' ) {
 	static $times = array();
 	if ( ! array_key_exists( $name, $times ) ) {
 		$times[ $name ] = microtime( true );
-		return;
+		return null;
```

Comment:
SonarLint suggests: "Refactor this function to use "return" consistently". An empty `return` is equivalent to `return null`, so this change should be safe.